Composer/packages/client/src/ShellApi.ts

```diff
@@ -301,6 +344,9 @@ export const ShellApi: React.FC = () => {
     apiClient.registerApi('removeLgTemplate', removeLgTemplateHandler);
     apiClient.registerApi('removeLgTemplates', removeLgTemplatesHandler);
     apiClient.registerApi('getLgTemplates', ({ id }, event) => getLgTemplates({ id }, event));
+    apiClient.registerApi('addLuIntent', addLuIntentHandler);
```

Comment:
where is get api?

Reply:
my expectation is through this set of api, form can update a whole lu intent, and a sub-section of lu intent

Reply:
> where is get api?

`luFiles` is accessible in `getState()` api, for current usage scenario seems no need to have another Fine-gained `get` api .

Reply:
> my expectation is through this set of api, form can update a whole lu intent, and a sub-section of lu intent



> my expectation is through this set of api, form can update a whole lu intent, and a sub-section of lu intent

get it, will make some change